alfresco-transformer-base/src/main/java/org/alfresco/transformer/util/RequestParamMap.java

```diff
@@ -72,4 +72,5 @@
     String INCLUDE_CONTENTS        = "includeContents";
     String NOT_EXTRACT_BOOKMARKS_TEXT = "notExtractBookmarksText";
     String PAGE_LIMIT              = "pageLimit";

```

Comment:
You will sen in my PR this static final really should be in the model project now.